lib/msf/ui/console/module_action_commands.rb

```diff
@@ -29,24 +29,42 @@ def commands
 
   #
   # Allow modules to define their own commands
+  # Note: A change to this method will most likely require a corresponding change to respond_to_missing?
   #
   def method_missing(meth, *args)
-    if (mod and mod.respond_to?(meth.to_s, true) )
+    if mod && mod.respond_to?(meth.to_s, true)
```

Comment:
Landed in master already: https://github.com/rapid7/metasploit-framework/pull/15738